### DISCUSSION TITLE
Translatable emails

### DISCUSSION THREAD

**Discussion by kevinmeijer97** [duncanmcclean/simple-commerce#766] [Help]
### What would you like to see added?
I would like to be able to translate the texts in the order emails.
Currently all texts are hardcoded in the email templates.
Even when you try to override them and add translations, the addon always sends the email from the default site.

### Is there a workaround you can use in the meantime?
Not at this time

### Priority wise - where's this feature for you?
We have a client that depends on having translatable emails.

**Comment by duncanmcclean** [comment]
> Even when you try to override them and add translations, the addon always sends the email from the default site.

Oh, Simple Commerce isn't using the locale of the site the order was made in? We should be doing that.

**Reply by kevinmeijer97** [reply]
That is correct, and it makes sense because there is no site data being saved on the order as far as i can see.
This addition would be great!

**Reply by duncanmcclean** [reply]
I'm currently off sick today but I'll take a look over the weekend! Thanks for pointing it out.

**Reply by duncanmcclean** [reply]
I've opened a bug report - if you subscribe to the issue, you'll get notified when it's fixed. #768
